komga-webui/src/locales/en.json

```diff
@@ -427,22 +427,22 @@
     },
     "filename_chooser": {
       "button_choose": "Choose",
-      "field_destination_filename": "Destination file name",
-      "label_source_filename": "Source File Name",
+      "field_destination_filename": "Destination filename",
+      "label_source_filename": "Source filename",
       "table": {
         "existing_file": "Existing File",
         "order": "Order"
       },
-      "title": "Destination File Name"
+      "title": "Destination Filename"
     },
     "password_change": {
       "button_cancel": "Cancel",
       "button_confirm": "Change password",
       "dialog_title": "Change password",
       "field_new_password": "New password",
-      "field_new_password_error": "New password is required.",
-      "field_repeat_password": "Repeat new password",
-      "field_repeat_password_error": "Passwords must be identical."
```

Comment:
Marked noun error. Should have been "The passwords …"

Reply:
What is a "Marked noun error." ? You keep using this term but i have no idea what this means.

Reply:
"Passwords" is in the air here. What passwords?
The generic capacity of passwords, or _the_ passwords.
While it isn't as bad here, this is how you introduce errors in translation.

Reply:
http://www.butte.edu/departments/cas/tipsheets/grammar/articles.html